app/src/androidTest/java/com/github/campus_capture/bootcamp/FirebaseBackendTest.java

```diff
@@ -154,6 +156,44 @@ public void onComplete(@NonNull Task<DataSnapshot> task) {
         }
     }
 
+    @Test
+    public void testVoteZoneInexistantZone()
+    {
+        // set database content
+        database.getReference().child("Users").child("testUserId").child("has_voted").setValue(false);
+
+        BackendInterface b = new FirebaseBackend();
+
+        try{
+            b.voteZone("testUserId", Section.IN, "zoneasdfajsdf").get();
+        }catch(Throwable e){
+            if(e.toString() != "Could not get result from the database"){
+                fail();
+            }
+
+        }
+    }
+    @Ignore
```

Comment:
I didn't manage to fix this test this sprint so I here is an issue to not forget it next sprint #154